### ISSUE TITLE
18.3 beta news?

### ISSUE THREAD

**Issue by Starboy-263** [leminlimez/Nugget#299]
Hey guys! Any chance of the new iOS version change gestalt options?

Hi from Brazil 🇧🇷

**Comment by chuck00000** [comment]
No, 18.3 beta 1 is very similar to 18.2 final version, nothing noticeable changes in that build…
